storage/innobase/xtrabackup/src/write_filt.cc

```diff
@@ -107,12 +107,17 @@ static bool wf_incremental_process(xb_write_filt_ctxt_t *ctxt,
   ulint page_size = cursor->page_size;
   byte *page;
   xb_wf_incremental_ctxt_t *cp = &(ctxt->wf_incremental_ctxt);
-
   for (i = 0, page = cursor->buf; i < cursor->buf_npages;
        i++, page += page_size) {
-    if (incremental_lsn > mach_read_from_8(page + FIL_PAGE_LSN)) {
+    /*
+     * PXB-2180 - for mysql.ibd we copy delta pages since the start
+     * of previous backup
+     */
```

Comment:
no need to mention bug number in the source.
Also, can you elaborate a bit on why use different LSN comparisons for mysql.ibd? 
How using this metadata_from_lsn solves the problem?
May be we can clarify that metadata_from_lsn is the LSN of last backup start?